src/composables/transaction.ts

```diff
@@ -99,18 +145,43 @@ export const useTransactionToken = (
     || getDexContracts.value.wae.includes(transaction.value.tx.contractId)
   ));
 
+  async function fetchOwnershipAccount(externalOwnerAddress: string | undefined) {
+    switch (ownershipStatus.value) {
+      case TRANSACTION_OWNERSHIP_STATUS.current:
+        return {
+          ...account.value,
+          label: i18n.t('transaction.overview.accountAddress'),
+          url: getExplorerPath.value(account.value.address),
+        };
+      case TRANSACTION_OWNERSHIP_STATUS.subAccount:
+        return accounts.value.find(
+          (acc) => [
+            transaction.value?.tx.accountId,
+            transaction.value?.tx.callerId,
+          ].includes(acc.address),
+        );
+      default: {
+        const address = externalOwnerAddress || txOwnerAddress.value;
+        const response = await fetchJson(`${activeNetwork.value.backendUrl}/profile/${address}`).catch(() => {});
+
+        return {
+          label: response?.preferredChainName || '',
```

Comment:
Why is this property names `label`? If this is a name. Is the new introduced property `label` is used anywhere else except for the places `something.name || something.label`, should it maybe be combined?